content/docs/user-guide/components/reference/shape/axis-aligned-box-shape.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Axis Aligned Box Shape component
+linktitle: Box
```

Comment:
Need to differentiate this from the box shape in the ToC.

```suggestion
 linktitle: Axis Aligned Box
```

Reply:
Change made.